### PR TITLE
Beacon accel

### PR DESCRIPTION
**Pull request by Svennixx**
Added needed config files to use Beacon Rev H build in accelerometer.

### CODE REVIEW

File: user_templates/printer.cfg [hunk 1]
```diff
@@ -200,6 +200,8 @@
 # [include config/hardware/accelerometers/adxl345_Fysetc_SB_Can_TH.cfg] # For ADXL plugged in Fysetc SB Can TH boards
 
 # [include config/hardware/accelerometers/lis2dw_usb_rp2040_spi1.cfg] # For BTT S2DW V1.0, ... 
+
+# [include config/hardware/accelerometers/eacon_accelerometer.cfg] # For Beacon Rev H make sure Beacon. Do not forget to include the beacon_virtual_probe in the PROBE section!
```
Comment (Frix-x) [inline user_templates/printer.cfg:204]: Be careful, there is a typo here in the file name: `eacon_accelerometer.cfg`

**Review by Frix-x — CHANGES_REQUESTED** [review]
Thanks for your PR, there's just a small typo

File: user_templates/printer.cfg (hunk 1 above)
Comment (Svennixx) [reply user_templates/printer.cfg]: Good catch, will fix this!

**Review by Svennixx — COMMENTED** [review]
Fixed typo

**Review by Frix-x — APPROVED** [review]

**Pull request merged**